frontend/src/component/Home/HomeNoWorkspace.jsx

```diff
@@ -1,10 +1,19 @@
 import React from 'react'
 import { translate } from 'react-i18next'
 
+import { IconButton } from 'tracim_frontend_lib'
+
 require('./Home.styl')
 
+const getWorkspaceMessage = (canCreate, canJoin) => {
+  if (canCreate && canJoin) return 'You can join an existing space or create your first space'
+  if (canCreate && !canJoin) return 'You can create your first space'
+  if (!canCreate && canJoin) return 'You can join an existing space'
+  return 'Please ask access to spaces to an administrator or space manager'
```

Comment:
if the string isn't wrapped in a t() function, it wont be parsed by the build-translation script.
To be clearer, your translation process works. It is only the translation keys that won't be added in the translation files by the script build-translation

Reply:
the solution is to call props.t on each strings of this functino instead of on the function return value

Reply:
I wasn't aware that we'd use such a script to detect translation keys, will change my code then.